target-platforms/target-platform-latest/target-platform-latest.target

```diff
@@ -13,7 +13,7 @@
     <location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
       <unit id="org.eclipse.ui.tests.harness" version="0.0.0"/>
       <unit id="org.eclipse.sdk.ide" version="0.0.0"/>
-      <repository location="https://download.eclipse.org/eclipse/updates/latest/"/>
+      <repository location="https://download.eclipse.org/eclipse/updates/4.35-I-builds/"/>
```

Comment:
Am I correct in guessing that this change means that the PR cannot be merged before 4.35 has been released?

Reply:
Yes, it'd need 4.35. Are LSP4E releases come out as new Eclipse being released? When does LSP4E adopt changes in the platform? When the new Eclipse is out already? Or a few months/weeks bore new Eclipse is out?